decoders/connector/mokosmart/LW006/v1.0.0/payload.js

```diff
@@ -0,0 +1,405 @@
+    return value;
+}
+
+function timezone_decode(tz) {
+    var tz_str = "UTC";
+    tz = tz > 128 ? tz - 256 : tz;
+    if (tz < 0) {
+        tz_str += "-";
+        tz = -tz;
+    } else {
+        tz_str += "+";
+    }
+
+    if (tz < 20) {
+        tz_str += "0";
+    }
+
+    tz_str += String(parseInt(tz / 2));
+    tz_str += ":"
+
+    if (tz % 2) {
+        tz_str += "30"
+    } else {
+        tz_str += "00"
+    }
+
+    return tz_str;
+}
+
+function parse_time(timestamp, timezone) {
+    timezone = timezone > 64 ? timezone - 128 : timezone;
+    timestamp = timestamp + timezone * 3600;
+    if (timestamp < 0) {
+        timestamp = 0;
+    }
+
+    var d = new Date(timestamp * 1000);
+    //d.setUTCSeconds(1660202724);
+
+    var time_str = "";
+    time_str += d.getUTCFullYear();
+    time_str += "-";
+    time_str += formatNumber(d.getUTCMonth() + 1);
+    time_str += "-";
+    time_str += formatNumber(d.getUTCDate());
+    time_str += " ";
+
+    time_str += formatNumber(d.getUTCHours());
+    time_str += ":";
+    time_str += formatNumber(d.getUTCMinutes());
+    time_str += ":";
+    time_str += formatNumber(d.getUTCSeconds());
+
+    return time_str;
+}
+
+function formatNumber(number) {
+    return number < 10 ? "0" + number : number;
+}
+
+String.prototype.format = function () {
+    if (arguments.length == 0)
+        return this;
+    for (var s = this, i = 0; i < arguments.length; i++)
+        s = s.replace(new RegExp("\\{" + i + "\\}", "g"), arguments[i]);
+    return s;
+};
+
+function signedHexToInt(hexStr) {
+    var twoStr = parseInt(hexStr, 16).toString(2); // 将十六转十进制，再转2进制
+    var bitNum = hexStr.length * 4; // 1个字节 = 8bit ，0xff 一个 "f"就是4位
+    if (twoStr.length < bitNum) {
+        while (twoStr.length < bitNum) {
+            twoStr = "0" + twoStr;
+        }
+    }
+    if (twoStr.substring(0, 1) == "0") {
+        // 正数
+        twoStr = parseInt(twoStr, 2); // 二进制转十进制
+        return twoStr;
+    }
+    // 负数
+    var twoStr_unsign = "";
+    twoStr = parseInt(twoStr, 2) - 1; // 补码：(负数)反码+1，符号位不变；相对十进制来说也是 +1，但这里是负数，+1就是绝对值数据-1
+    twoStr = twoStr.toString(2);
+    twoStr_unsign = twoStr.substring(1, bitNum); // 舍弃首位(符号位)
+    // 去除首字符，将0转为1，将1转为0   反码
+    twoStr_unsign = twoStr_unsign.replace(/0/g, "z");
+    twoStr_unsign = twoStr_unsign.replace(/1/g, "0");
+    twoStr_unsign = twoStr_unsign.replace(/z/g, "1");
+    twoStr = -parseInt(twoStr_unsign, 2);
+    return twoStr;
+}
+
+
+function getData(hex) {
+    var length = hex.length;
+    var datas = [];
+    for (var i = 0; i < length; i += 2) {
+        var start = i;
+        var end = i + 2;
+        var data = parseInt("0x" + hex.substring(start, end));
+        datas.push(data);
+    }
+    return datas;
+
+}
+function addPayloadArray(variable, value) {
+    return payload.push({ variable: variable, value: value, group: String(payload_raw.group), serie: String(payload_raw.serie), });
```

Comment:
`serie` is deprecated. This payload parser must be updated to use `group` instead